### ISSUE TITLE
Cant get forms to work inside areas

### ISSUE THREAD

**Issue by JasonRodman** [valdisiljuconoks/MvcAreasForEPiServer#5]
For some reason the area specific form element block view locations are not being registered. Loaded your nuget package after successfully implementing the base areas support package, but form element templates only work when in the root shared folder. Ones in the views folders are ignored. In developer tools I checked the view locations and the area specific view locations are not there. Even went as far as copying the view engines code and implemented it manually, verified it runs when loading, but still does not work. I am also using the EPiServerBootstrapArea and corresponding forms package, could that be conflicting with this?

**Comment by valdisiljuconoks** [comment]
Hi,

So the case (just to get an idea and understand more) is that you do have:
* display templates for form elements in root Views/ folder. and that is working as expected
* you do have some areas defined and have page types there
* you have installed this package
* view locations in areas are not registered in view engines

Just trying to understand your setup. Ideally if you could drop not working AlloyTech sample project. That would make things go faster.

**Comment by JasonRodman** [comment]
Yes, that is the case. The form element templates from the areas are not registered. I confirmed this in both developer tools and by removing the root views/shared form element templates. They revert back to the built in ones. I wish I had the time to repro in an alloy demo site for you. Unfortunately deadlines are approaching so I have to work around it. At the moment I have just left it in the root folder for now in hopes that you can find a solution. It will become more important once I have additional sites with different form element styles that require them to be edited. Can you try putting both your bootstrap content area and the MVC areas packages into the same project, I suspect they are conflicting with each other.
